corehq/ex-submodules/phonelog/reports.py

```diff
@@ -241,24 +241,25 @@ def _create_row(self, log, matching_id, _device_users_by_xform, user_query, devi
         if len(tag_classes) == 1:
             tag_classes.append('label-info')
 
-        log_tag_format = self._log_tag_fmt % {
-            "url": "%s?goto=%s" % (self.get_url(domain=self.domain),
-                                   html.escape(json.dumps(log.id))),
-            "classes": " ".join(tag_classes),
-            "text": log_tag,
-            "extra_params": (' data-datatable-highlight-closest="tr"'
-                             if log.id == matching_id else ''),
-            "tooltip": "Show the surrounding 100 logs."
-        }
+        log_tag_format = format_html(
+            self._log_tag_fmt,
+            url="%s?goto=%s" % (self.get_url(domain=self.domain),
+                html.escape(json.dumps(log.id))),
+            classes=" ".join(tag_classes),
+            text=log_tag,
+            extra_params=(' data-datatable-highlight-closest="tr"' if log.id == matching_id else ''),
+            tooltip="Show the surrounding 100 logs."
```

Comment:
Maybe out of scope for this PR, but shouldn't this be translated?

Reply:
Translation would make sense. But yes, the scope of this PR is essentially just finding all the places where data tables aren't sanitizing their output and updating them. I'd generally agree with going in and making small changes to leave the code better than I found it, but I think you can see, with the amount of changes required all over the place, that I want to keep the scope as narrow as possible here.